python-threatexchange/threatexchange/hashing/pdq_faiss_matcher.py

```diff
@@ -209,9 +255,23 @@ def mih_index(self):
             return faiss.downcast_IndexBinary(self.faiss_index.index)
         return self.faiss_index
 
-    def search(self, queries: t.Sequence[PDQ_HASH_TYPE], threshhold: int, **kwargs):
+    def search(
+        self,
+        queries: t.Sequence[PDQ_HASH_TYPE],
+        threshhold: int,
+        return_as_ids: bool = False,
+    ):
+        self.mih_index.nflip = threshhold // self.mih_index.nhash
+        return super().search(queries, threshhold, return_as_ids)
```

Comment:
What's the superclass in this case? Isn't is ABC/object at this point?

Reply:
this was just the removal of **kwargs
I'll remove this change to address later.